setup.py

```diff
@@ -8,9 +8,24 @@
 """
 
 import re
+import sys
 
 from setuptools import setup, find_packages
 
+def _get_requirements(path):
```

Comment:
Manual parsing of `requirement.txt` is not a very good idea - it won't work in many cases, e.g.:
```
docker-py < 2.0  #New docker-py has a totally different API
```